scripts/docker/build-debian7/Dockerfile.deps

```diff
@@ -1,5 +1,8 @@
 FROM debian:wheezy
 
+# Fix debconf warnings upon build
```

Comment:
No need to comment what this is, just add the ARG (next to the other ARGs there if there are some).

It's obvious what it does (and should have been there before :-) thanks.)

Reply:
OK, let me update the patch.

Reply:
Done, just refresh the commit @mcnewton

Reply:
Ta